web/cypress/integration/route/search-route.spec.js

```diff
@@ -126,9 +135,35 @@ context('Create and Search Route', () => {
     cy.contains(data.test2).should('not.exist');
   });
 
+  it('should search the route with path', function () {
+    cy.visit('/');
+    cy.contains('Route').click();
+    // full match
+    cy.get(selector.pathSearch).type(data.uris1);
+    cy.contains('Search').click();
+    cy.contains(data.uris1).should('contain', data.uris1);
+    cy.contains(data.uris0).should('not.exist');
+    cy.contains(data.uris2).should('not.exist');
+    // partial match
+    cy.reload();
```

Comment:
Hi @iamayushdas , may I ask why we need `cy.reload` here? It seems that we don't need to reload the page, If we only want to change the path for search param, shall we use e.g. `selector.clear().type('new path')`?

Reply:
I was following the legacy of above name search test, You can see that in name search also we have used `cy.reload()`
just to reload the screen to refresh
![Screenshot from 2021-05-31 21-27-42](https://user-images.githubusercontent.com/40708551/120218428-58a3b880-c257-11eb-8473-3a7730d2057a.png)

Reply:
should i change name search part too?

Reply:
got it @iamayushdas , we can optimize this point in a new pr

Reply:
okay, Do you want me to give another PR?

Reply:
> okay, Do you want me to give another PR?

I just created an issue: https://github.com/apache/apisix-dashboard/issues/1925, pr is welcome ^^